pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1444,6 +1444,12 @@ self: super: {
   # So let's not go there any just disable the tests altogether.
   hspec-core = dontCheck super.hspec-core;
 
+  # Missing a file required for tests in 0.4, fixed in 0.4.1
+  # Also use correct dependency
```

Comment:
Is it possible to just wait until nixpkgs is updated to autoapply-0.4.1?

Also, we usually like to target the `haskell-updates` branch for Haskell-related changes.

Reply:
Thanks, 

It'll still need to use th-desugar-1.11. I'll close this PR and open a new one with just that change targeting the haskell-updates branch when nixpkgs is using 0.4.1